src/components/webview/DiffWebviewPanel.ts

```diff
@@ -77,7 +77,10 @@ export class DiffWebviewPanel extends IntuitaWebviewPanel {
 		}
 
 		const job = this.__jobManager.getJob(jobHash);
-		const jobAccepted = this.__jobManager.isJobAccepted(jobHash);
+		// @TODO
```

Comment:
why's there a TODO?

Reply:
I left todo because i did not removed all code that was depending on `jobAccepted`